### PR TITLE
Issue-Fix-#819 append: toString methods

### LINKED ISSUE #819: Add toString() methods for regularly logged entities
- [ ] User -> print id and username
- [ ] Game -> print id and title

### PR DESCRIPTION
**Pull request by l3002**
Fixes #819 

## Description

Overrode toString() method for following classes:

- [x] User - prints ID and Email (wasn't able to find the username for this class therefore used email instead)
- [x] Game - prints ID and Title

### CODE REVIEW

File: src/main/java/com/faforever/api/data/domain/User.java [hunk 1]
```diff
@@ -19,4 +19,10 @@ public class User extends Login {
   public String getPassword() {
     return password;
   }
+
+  //Overriding toString method for efficient Logging
```
Comment (Brutus5000) [inline src/main/java/com/faforever/api/data/domain/User.java:23]: Is there a particular reason not to use Lombok here?

**Review by Brutus5000 — CHANGES_REQUESTED** [review]
Please just use the variable names as is. I know I wrote title in the issue description, but I wasn't aware of the variables name.

File: src/main/java/com/faforever/api/data/domain/User.java (hunk 1 above)
Comment (l3002) [reply src/main/java/com/faforever/api/data/domain/User.java:23]: Yeah, If we use the ToString Annotation for User class then that annotation will have to call the super classes toString methods recursively as the User class inherit those properties from the classes up in the heirarchy, which leads to a messy format. Something like this User(super(super(id = 1232)), super(email = xyz@gmail.com)).

This is the reason why I used the traditional way of overriding the toString() method

Comment (Brutus5000) [reply src/main/java/com/faforever/api/data/domain/User.java:23]: ok. But please make the fields names lowercase :)

Comment (l3002) [reply src/main/java/com/faforever/api/data/domain/User.java:23]: @Brutus5000  I've added two new commits to this PR. Could you please check if the naming convention is alright now?

**Review by Brutus5000 — APPROVED** [review]

**Pull request merged**